### ISSUE TITLE
Add openssh to rstudio-session-complete image

### ISSUE THREAD

**Issue by TheisFerre** [rstudio/rstudio-docker-products#457]
Could you add openssh to the rstudio-session-complete images. This would allow users to create SSH keys and it is also required for git clone.

i.e. add the following to the bionic image:

```dockerfile
RUN apt-get update && apt-get install -y openssh-server 
```

**Comment by colearendt** [comment]
This is an interesting proposal, thanks! This definitely seems useful. I suspect we would only want to install the client utilities, not the full "openssh-server" 😄

**Comment by TheisFerre** [comment]
Yeah, that makes perfect sense. Thanks for the response @colearendt!

**Comment by colearendt** [comment]
Related to #503

**Comment by bschwedler** [comment]
`openssh-client` is installed the base image
https://github.com/rstudio/rstudio-docker-products/blob/94a3f81cb31da2d7fbc55d696a7644d59d1bba26/product/base/deps/ubuntu2204_packages.txt#L67
